target/linux/lantiq/image/Makefile

```diff
@@ -714,6 +729,30 @@ define Device/arcadyan_vgv7519-nor
 endef
 TARGET_DEVICES += arcadyan_vgv7519-nor
 
+define Device/arcadyan_vgv952cjw33-e-ir
+  $(Device/NAND)
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := VGV952CJW33-E-IR
+  DEVICE_ALT0_VENDOR := Vodafone
+  DEVICE_ALT0_MODEL := Easybox 904xDSL
+  DEVICE_DTS := VGV952CJW33-E-IR
+  SUPPORTED_DEVICES += VGV952CJW33-E-IR
+  IMAGES := sysupgrade.bin
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | lzma | uImage lzma | pad-offset 16 0 | append-uImage-dummyrootfs
+endef
+
+define Device/arcadyan_vgv952cjw33-e-ir-vpe
+  $(Device/arcadyan_vgv952cjw33-e-ir)
```

Comment:
Please do not derive one device from another. Create a shared definition which is then inherited by both, like for the fritz 3370:
https://github.com/openwrt/openwrt/blob/master/target/linux/lantiq/image/vr9.mk#L99